index.html

```diff
@@ -0,0 +1,136 @@
+      <h2 class="section-title text-center">Main Programs</h2>
+      <hr class="section-dash-splitter">
+      <section class="programs full-with d-flex  cols wrap">
+          <div class="d-flex rows program">
+            <img src="images/program1.png" alt="Lecture">
+            <h3>Lecture</h3>
+            <p>
+              Lorem ipsum dolor sit amet consectetur adipisicing elit.
+            </p>
+          </div>
+          <div class="d-flex rows program">
+            <img src="images/program2.png" alt="CC Exhibition">
+            <h3>CC Exhibition</h3>
+            <p>
+              Lorem ipsum dolor sit amet consectetur adipisicing elit.
+            </p>
+          </div>
+          <div class="d-flex rows program">
+            <img src="images/program3.png" alt="Forum">
+            <h3>Forum</h3>
+            <p>
+              Lorem ipsum dolor sit amet consectetur adipisicing elit.
+            </p>
+          </div>
+          <div class="d-flex rows program">
+            <img src="images/program4.png" alt="Workshop">
+            <h3>Workshop</h3>
+            <p>
+              Lorem ipsum dolor sit amet consectetur adipisicing elit.
+            </p>
+          </div>
+          <div class="d-flex rows program">
+            <img src="images/program5.png" alt="CC Ignite">
+            <h3>CC Ignite</h3>
+            <p>
+              Lorem ipsum dolor sit amet consectetur adipisicing elit.
+            </p>
+          </div>
+      </section>
+      <a href="#" class="join-btn d-flex"> JOIN CC Global Summit 2015 </a>
+    </section>
+
+    <section class="featured-speakers d-flex wrap cols">
+      <h2 class="section-title text-center">Featured Speakers</h2>
+      <hr class="section-dash-splitter">
+      <div class="d-flex featured-speakers-wrapper cols">
+        
+      </div>
+    </section>
+    <button class="d-fles rows j-center see-more-btn mobile-only">
+      More <img src="images/arrow-204-24.png" alt="Click here to see more...">
+    </button>
+    <section class="desktop-only partners d-flex cols">
+      <h2 class="section-title text-center">partners</h2>
+      <hr class="section-dash-splitter">
+      <div class="d-flex rows wrap s-evenly partners-logos ">
+        <img src="images/partner1.png" alt="Mozilla">
+        <img src="images/partner2.png" alt="Google">
+        <img src="images/partner3.png" alt="Naver">
+        <img src="images/partner4.png" alt="Kakao">
+        <img src="images/partner5.png" alt="AirBnB">
+      </div>
+    </section>
+  </main>
+
+  <footer>
+    <div class="footer d-flex rows s-between light">
+      <img src="images/compoundcode-logo-1.png" alt="CompoundCode">
+      <section>
+        <h4>&copy; 2021 CompoundCode. All rights reserved</h4>
+        <p>Developed and designed by <a href="https://github.com/blessedjasonmwanza">Blessed Jason Mwanza</a></p>
+      </section>
+    </div>
+    <script src="js/main.js"></script>
+  </footer>
```

Comment:
- [ ] For mobile size Footer should have a white background. Currently, it is not a good one, you can check the design below. Let's fix that.
- [ ]  You may add some margin as well between the items so that it feels more spaced. That will help to have a better UX.

![Screenshot from 2021-10-14 08-17-05](https://user-images.githubusercontent.com/40503704/137269772-1f8a000b-a91d-4455-b8d7-4ead6e42401e.png)

![Screenshot from 2021-10-14 08-14-19](https://user-images.githubusercontent.com/40503704/137269862-fb20e533-7e47-43b7-95e0-5b69af070a6f.png)

- [ ] For desktop size Footer should have a dark background. Currently, it is not a good one, you can check the design below. Let's fix that.

![Screenshot from 2021-10-14 08-21-52](https://user-images.githubusercontent.com/40503704/137270456-1f7dce4a-759f-4ab6-9e4d-9901db703cf8.png)
![Screenshot from 2021-10-14 08-20-38](https://user-images.githubusercontent.com/40503704/137270512-716d7ad9-c13c-48db-913b-b5d2e813b11a.png)